packages/country-identity-kit/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "country-identity-kit",
+  "version": "0.0.1",
+  "description": "Frontend component kit for the country identity pcd package",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsup src/index.ts --dts",
+    "dev": "tsup src/index.ts --dts --watch",
+    "lint": "eslint --ext .js,.ts .",
+    "format": "prettier --ignore-path .prettierignore --write \"**/*.+(js|ts|json)\"",
+    "test": "TS_NODE_COMPILER_OPTIONS={\\\"module\\\":\\\"commonjs\\\"} npx ts-mocha -p tsconfig.json test/**/*.test.{ts,tsx} --exit",
+    "prepare": "husky install",
+    "pre-publish": "yarn build && yarn lint && yarn test && npm publish --dry-run",
+    "npm-publish": "yarn build && yarn lint && yarn test && npm publish"
+  },
+  "files": [
+    "./src/*",
+    "./dist/*",
+    "./README.md",
+    "tsconfig.json",
+    "package.json"
+  ],
+  "author": "Yanis Meziane for PSE",
+  "license": "MIT",
+  "dependencies": {
+    "@pcd/pcd-types": "^0.6.0",
+    "@peculiar/x509": "^1.9.3",
+    "node-forge": "^1.3.1",
+    "pcd-country-identity": "^0.0.5",
```

Comment:
I think we can use "pcd-country-indentity: workspace:*" here

Reply:
I'm going to check on that!